dataclients/kubernetes/definitions/ingressv1.go

```diff
@@ -101,15 +101,10 @@ func ValidateIngressesV1(ingressList IngressV1List) error {
 			name := i.Metadata.Name
 			namespace := i.Metadata.Namespace
 			nerr = fmt.Errorf("%s/%s: %w", name, namespace, nerr)
-			err = errors.Wrap(err, nerr.Error())
+			err = errorsJoin(err, nerr)
```

Comment:
Lets update `// ValidateIngresses is a no-op` comment as its definitely not a no-op

Reply:
It's actually no-op

Reply:
`ValidateIngressV1` always return `nil`.

Reply:
This also should collect errors in a slice first.

Reply:
Please fix the comment to `// ValidateIngressesV1 validates an IngressV1List` or simply remove it

Reply:
This should be resolved